demokratie-live/democracy-client
# corrected open icon to align with text

**Issue:** #209

**Description:**

Align Open Icon with Text

**How to test:**

- Open Screen "Schon Gewusst"
- Check Collapse Icon
- Is it on the same height as the text?

**Checklist:**
